backend/src/model/model.py

```diff
@@ -0,0 +1,392 @@
+import json
+import operator
+import numpy as np
+import pandas as pd
+
+import spotipy
+from spotipy.oauth2 import SpotifyClientCredentials
+
+from tqdm import tqdm
+
+from sklearn.preprocessing import MinMaxScaler
+from sklearn import mixture
+from sklearn.model_selection import train_test_split
+from sklearn.neighbors import KNeighborsClassifier
+
+import pickle
+import os
+
+
+# abspath = os.path.abspath(__file__)
+# dname = os.path.dirname(abspath)
+# os.chdir(dname)
+
+
+def continue_playlist(my_uris):
+    
+    if len(my_uris) == 0:
+        raise ValueError("At least 1 track_uri must be provided")
+    
+    # define global variables
+    SIZE = 500  # this defines how many playlists we want
+    
+    # for Spotify API
+    cid = 'cc9a4cae89f3487f99ab84b46002abea'
+    secret = 'd8d0ed623ff4428893a0d4793a764402'
+    client_credentials_manager = SpotifyClientCredentials(client_id=cid, client_secret=secret)
+    sp = spotipy.Spotify(client_credentials_manager=client_credentials_manager)
+    
+    def unpack(json_name):
+        '''
+        unpack a json playlist file to obtain a playlist
+        input: a file name
+        return: SIZE list of playlists
+        '''
+        # Opening JSON file
+        f = open(json_name)
+        
+        # returns JSON object as
+        # a dictionary
+        data = json.load(f)
+        f.close()
+        playlists = data['playlists']
+        return playlists
+    
+    playlists = unpack(json_name='src/model/data/playlists.json')
+    
+    # print(playlists[0]["tracks"][0])
+    
+    def find_uris(playlists, start=0, SIZE=SIZE):
+        '''
+        ouput the uri list for a given playlist
+        input: a playlist
+        return: a list of all track uris, and the playlist uri
+        '''
+        track_uris = [[i['track_uri'] for i in playlists[j]['tracks']] for j in range(start, SIZE)]
+        pids = [playlists[i]['pid'] for i in range(start, SIZE)]
+        return track_uris, pids
+    
+    # check track_uris for first playlist in dataset
+    start = 0
+    uri_list, pids = find_uris(playlists, start, SIZE)
```

Comment:
There are various parts of the code that are currently unused because the current model is a simplified version of the full one. Stuff like that will be used in the final production version of the model.